roles/grafana/templates/dashboards.yaml.j2

```
@@ -0,0 +1,11 @@
+apiVersion: 1
+
+providers:
+  - name: 'Dashboards'
+    type: file
+    disableDeletion: false
+    updateIntervalSeconds: 10
+    allowUiUpdates: false
+    options:
+      path: {{ grafana__dashboards_path | to_yaml }}
```

In my system, a spurious blank line is introduced here. It goes away with:

```suggestion
      path: {{ grafana__dashboards_path | to_yaml | trim }}
```

(This suggestion is 400% optional.)

---

Done!